tests/test_self.py

```diff
@@ -233,7 +233,6 @@ def test_parallel_execution(self) -> None:
                 join(HERE, "functional", "a", "arguments.py"),
             ],
             out=out,
-            # We expect similarities to fail and an error
```

Comment:
No reason to expect similarity to fail when checking only one file. Maybe an earlier version of this test checked the same file twice?

Reply:
The comment was very unclear, sorry about that.

It was meant to say something like:

> if we test the same file twice, which this test _was_ doing, in parallel mode we get extra errors, so we only test a single file here in parallel mode, ensuring it errors. All parallel checks are in their own file now.